src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/db/ScriptManagerImpl.java

```diff
@@ -73,12 +158,41 @@ public void executeScript(
     }
   }
 
+  private String getUtcTimeStringFromTimestamp(Timestamp timestamp) {
+    Instant instant = timestamp.toInstant();
+    // Replace the "Z" that signifies UTC timezone with "S" that signifies the separator between
+    // seconds and fractional seconds; remove separators that are not necessary for human
+    // interpretation.
+    String instantWithoutNanoseconds =
+        instant
+            .truncatedTo(ChronoUnit.SECONDS)
+            .toString()
+            .replaceAll("[\\-:]", "")
+            .replaceAll("[Z]", "S")
```

Comment:
The toString() method intrinsically uses DateTimeFormatter.ISO_INSTANT, which is both timezone-agnostic and locale-agnostic, thus I prefer using that one. I modified the string substitution so that only one regex scan is needed now.